docs/_static/auth/iam.yaml

```diff
@@ -535,15 +571,17 @@ paths:
           description: |
             This is returned when an OAuth2 Client is unauthorized to talk to the introspection endpoint.
             Note: introspection of an invalid or malformed token returns a 200 where with field 'active'=false
-  /iam/{id}/statuslist/{page}:
+  /oauth2/{did}/statuslist/{page}:
```

Comment:
This is not an oauth related endpoint. Change base path and maybe also put it in a different yaml

Reply:
Moved to `/statuslist/<did>/<page>`